deployment/README.md

```diff
@@ -0,0 +1,10 @@
+To deploy this run 
+
+```
+make up-stage
+```
+
+and the way this was built initially was using [kompose](https://kompose.io/)
```

Comment:
Is this information valuable ?

Reply:
This is important in case someone wants to re-generate, can be removed but I thought info is power